### PR TITLE
format: HEP brief format fixes

### PR DESCRIPTION
**Pull request by nikpap**
* Fixes arrow to show more abstract.

* Display number of references.

Signed-off-by: Nick Papoutsis <admin@nils.gr>

### CODE REVIEW

File: inspire/base/static/less/format/brief-results.less [hunk 1]
```diff
@@ -20,9 +20,9 @@
  */
 
 
-@import (reference) "../inspire.less";
+ @import (reference) "../inspire.less";
```
Comment (jmartinm) [inline inspire/base/static/less/format/brief-results.less]: why this extra space in front?

File: inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl [hunk 2]
```diff
@@ -133,21 +132,33 @@
...
+        <div class="citations-references">
+
         {% if record.get('date_updated') %}
-        <i class="glyphicon glyphicon-calendar"></i> {{ record.get('date_updated').split('-')[0] }}<br/>
+          <i class="glyphicon glyphicon-calendar"></i> {{ record.get('date_updated').split('-')[0] }}<br/>
         {% endif %}               
+
         {% if  record.get('_cited_by_count') > 0  %}
-        <i class="fa fa-quote-left"></i><span><a href="/record/{{ record.get('_id') }}/citations"  target="_blank"> Cited {{ record.get('_cited_by_count') }} times</a></span><br/>
+
```
Comment (jmartinm) [inline inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: a lot of whitespace here. can you remove it?

File: inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl [hunk 3]
```diff
@@ -29,9 +29,11 @@
   {% endfor %}
   {% if record.authors|length > number_of_displayed_authors %}
   {{ sep() }}
-  <a href="#authors_{{ record['recid'] }}" class="text-muted" data-toggle="modal" data-target="#authors">
-      <em>{{ _(' Show all') }}</em>
-  </a>
+  <small>
+    <a href="#authors_{{ record['recid'] }}" class="text-muted" data-toggle="modal" data-target="#authors">
+        <em>{{ _(' Show all') }}</em>
```
Comment (jmartinm) [inline inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl]: was the `Show all` link made more consistent in size with the authors as we mentioned this morning?

File: inspire/base/static/less/format/brief-results.less (hunk 1 above)
Comment (nikpap) [inline inspire/base/static/less/format/brief-results.less]: Has been removed.

File: inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl (hunk 2 above)
Comment (nikpap) [inline inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: Has been removed.

File: inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl (hunk 3 above)
Comment (nikpap) [inline inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl]: Yes, it's now <small>, in line with the author's size.

File: inspire/base/static/less/format/brief-results.less [hunk 4]
```diff
@@ -63,6 +63,10 @@ hr {
     cursor:pointer;
 }
 
+.citations-references {
```
Comment (jmartinm) [inline inspire/base/static/less/format/brief-results.less]: here there is still some space in front. I think its the last thing to fix.

**Comment by jmartinm** [comment]
Merged 693a25cb7e8f8de3da78b51077775beb4f02faae

**Pull request closed without merging**